src/main/java/seedu/address/logic/commands/EmailCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Command that copies the emails of the persons in the current list to clipboard, with comma separations.
+ */
+public class EmailCommand extends Command {
+    public static final String COMMAND_WORD = "email";
+
+    public static final String MESSAGE_SUCCESS = "Copied emails to clipboard";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        StringBuilder emails = new StringBuilder();
+        for (Person p : model.getPersonList()) {
```

Comment:
I am wondering if we could SLAP more here. Perhaps sth like:
```
String emails = collectEmails(model);

private String collectEmails(Model model) {...}
```
Let me know what you think:)

Reply:
Sounds good, changes have been made.